operations/kubernetes/Cert_Renewal_for_Kubernetes_and_Bare_Metal_EtcD.md

```diff
@@ -1,22 +1,21 @@
 # Kubernetes and Bare Metal EtcD Certificate Renewal
 
-## Scope
-
 As part of the installation, Kubernetes generates certificates for the required subcomponents. This document will help walk through the process of renewing the certificates.
 
-**`IMPORTANT:`** Depending on the version of Kubernetes, the command may or may not reside under the alpha category. Use `kubectl certs --help` and `kubectl alpha certs --help` to determine this. The overall command syntax should be the same and this is just whether or not the command structure will require `alpha` in it.
-
-**`IMPORTANT:`** When you pick your master node to renew the certificates on, that is the node that will be referenced in this document as `ncn-m`.
+**IMPORTANT:**
 
-**`IMPORTANT:`** This document is based off a base hardware configuration of 3 masters and 3 workers (We are leaving off utility storage because they are not running Kubernetes). Please make sure to update any commands that run on multiple nodes accordingly.
+- Depending on the version of Kubernetes, the command may or may not reside under the `alpha` category. Use `kubectl certs --help` and `kubectl alpha certs --help` to determine this.
+  The overall command syntax should be the same and this is just whether or not the command structure will require `alpha` in it.
+- The node referenced in this document as `ncn-m` is the master node selected to renew the certificates on.
+- This document is based off a base hardware configuration of three master nodes and three worker nodes. Utility storage nodes are not mentioned because they are not running Kubernetes. Please make sure to update any commands that run on multiple nodes accordingly.
```

Comment:
```suggestion
- This document is based off a base hardware configuration of three master nodes and three worker nodes. Utility storage nodes are not mentioned because they are not running Kubernetes. Make sure to update any commands that run on multiple nodes accordingly.
```